src/vuestic-theme/vuestic-components/va-table/VaTable.vue

```diff
@@ -0,0 +1,215 @@
+<template>
```

Comment:
You may want to rename this to `VaDataTable` as `.va-table` is already reserved for global style.